web/viewer.css

```diff
@@ -146,6 +150,7 @@ select {
   top: 32px;
   bottom: 0;
   width: 200px;
+  width: var(--sidebar-width);
```

Comment:
For one of these occurrences, can you add a comment why `width` is duplicated? I think it's done so that if browsers don't support CSS variables they fall back to the old `200px` width, but I can imagine that this CSS looks a bit weird if it's not known.

Reply:
> I think it's done so that if browsers don't support CSS variables they fall back to the old 200px width,

That's correct!
I've added a comment to the first of these occurrences, please check if you think clear enough.